EIPS/eip-creation_collision.md

```diff
@@ -0,0 +1,74 @@
+---
+eip: <not assigned>
+title: Revert creation in case of collision
+description: Revert contract creation if address already has code
+author: Renan Rodrigues de Souza (@RenanSouza2)
+discussions-to: https://ethereum-magicians.org/t/eip-revert-on-address-collision/13442
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-03-20
+---
+
+## Abstract
+
+There is no current definition to what happens when a contract creation happens in an address with code already deployed. This fix prevents an attack of deploying a contract code and later changing the code arbitrarily.
```

Comment:
```suggestion
There is no current definition to what happens when a contract creation happens in an address with code already deployed. This fix prevents an attack of deploying contract code and later changing the code arbitrarily.
```

Reply:
Thank you for the remarks, I applied them